python/Ganga/Lib/Localhost/LocalHostExec.py

```diff
@@ -75,8 +75,7 @@
 import tarfile
 
 fullenvironment = os.environ.copy()
-for key,value in environment.iteritems():
-    fullenvironment[key] = value
+fullenvironment['PATH'] = os.getcwd():fullenvironment['PATH']
```

Comment:
This looks like a typo (`:fullenvironment['PATH']`) unless I'm misunderstanding something.